burner-ui/src/Pages/ConfirmPage/ConfirmPage.tsx

```diff
@@ -30,14 +31,27 @@ const ConfirmPage: React.FC<BurnerContext & RouteComponentProps> = ({
     id
   } = history.location.state;
   const [asset] = assets.filter(a => a.id === assetId);
+  console.log(history.location.state);
 
-  const amount = ether || asset.getDisplayValue(value);
+  // HAVE TO DO THIS TO GET THE VALUE TO BE PASSED
+  // NEEDS FIX
+  const newVal = Math.round(value);
```

Comment:
Math.round is in here to prevent the error — it doesn't like anything with a decimal place. The raw value itself doesn't work, but maybe that's because I'm passing a decimal value, rather than a wei-like value? I'm unsure of how to change this.

Reply:
Math.round is in here to prevent the error — it doesn't like anything with a decimal place. The raw value itself doesn't work, but maybe that's because I'm passing a decimal value, rather than a wei-like value? I'm unsure of how to change this.